tests/unit/test_base_path.py

```diff
@@ -0,0 +1,183 @@
+import requests
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+
+auth = UsernamePasswordAuth('username', 'password')
+
+"""
+#This is a test file for testing imperfect base paths, and various other tests too.
+"""
+
+"""
+#The first five tests demonstrate that the client can handle badly written base paths and produce correct final urls.
+#Each test runs a request to the server, and produces the correct final url.
+"""
+
+
+@responses.activate
+def test_base_path_no_trailing_slashes():
```

Comment:
There is only one trailing slash, therefore `_no_trailing_slash` rather than `_no_trailing_slashes`